core/aggregation.go

```diff
@@ -132,7 +157,19 @@ func (a *StdSignatureAggregator) ReceiveSignatures(ctx context.Context, state *I
 
 	for numReply := 0; numReply < numOperators; numReply++ {
 		var err error
-		r := <-messageChan
+
+		var r SigningMessage
+
+		select {
+		case r = <-messageChan:
+		case <-attestationCtx.Done():
+			remainingReplies := numOperators - numReply
+			a.Logger.Warnf(
+				"batch attestation time exceeded, no further signatures will be accepted for batch %x. "+
+					"There are %d uncollected signatures.", message, remainingReplies)
+			break
+		}
```

Comment:
Note to the reviewers: this is the primary change in this PR. Most of the rest of the diff is fixing line wrapping in these files (I find code harder to read with really long lines).